modules/lobbies/scripts/list.ts

```diff
@@ -29,8 +32,16 @@ export async function run(
 	ctx: ScriptContext,
 	req: Request,
 ): Promise<Response> {
-	// TODO: Cache this without hitting the DO
+	await ctx.modules.captcha.guard({
+		key: "lobbies.list",
+		period: 20,
+		requests: 5,
+		type: "default",
```

Comment:
type & key are flipped here iirc if we want to keep this consistent with rate limit. can chat about a better api for this over slack.